app/views/stats/index.json.jbuilder

```diff
@@ -0,0 +1,22 @@
+json.teaching_jobs do
+  json.jobs do
+    json.published @published
+    json.scraped @scraped
+    json.draft @draft
+    json.trashed @trashed
+  end
+
+  json.sign_ins do
+    json.authenticated @signin_audit['dfe-sign-in.authentication.success']
+    json.authorised @signin_audit['dfe-sign-in.authorisation.success']
+    json.unauthorised @signin_audit['dfe-sign-in.authorisation.failure']
+  end
+
+  if @audit.present?
+    json.audit @audit do |audit|
+      json.partial! 'show.json.jbuilder', audit: audit
```

Comment:
We said we would display only a summary of information (in JSON) as that's all that is needed by the people who need to consume it, unsure why we are now surfacing all information we hold instead. This does now begin to explain why authentication has been added in.